src/index.tsx

```diff
@@ -13,6 +14,7 @@ root.render(
       <Routes>
         <Route path="/" element={<App />} />
         <Route path="/home" element={<Home />} />
+        <Route path="/createAccount" element={<CreateAccount />} />
```

Comment:
NON-BLOCKING: Why don't we rename this `/register` since that's what the API endpoint is called also?

Reply:
Done!

Reply:
🎉